### PR TITLE
sperrorest blog post

### PR DESCRIPTION
**Pull request by pat-s**
Hi Edzer et al.,

here is our blog post regarding the major update of `sperrorest`.
I included a .bib file for the references in the folder `bibs`.  Feel free to move it if the place/folder name is not appropriate. 
If any modifications are needed, please let me know. I'm unsure about the rendered output size of the image in the final .md file - happy to adjust it if it does not fit that good with the layout. 

Best, Alex & Patrick

### CODE REVIEW

File: _rmd/2017-03-08-sperrorest-update.Rmd [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+
+Nevertheless, this increased flexibility comes with the risk of possible over-fitting to the training data. Since nearby spatial observations often tend to be more similar than distant ones, traditional random cross-validation is unable to detect this over-fitting whenever spatial observations are close to each other (e.g. @Brenning2005). Spatial cross-validation addresses this by resampling the data not completely randomly, but using larger spatial regions. In some cases, spatial data is grouped, e.g. in remotely-sensed land use mapping grid cells belonging to the same field share the same management procedures and cultivation history, making them more similar to each other than to pixels from other fields with the same crop type.
+
+This package provides a customizable toolkit for cross-validation (and bootstrap) estimation using a variety of spatial resampling schemes. More so, this toolkit can even be extended to spatio-temporal data or other complex data structures. This vignette will walk you through a simple case study, crop classification in central Chile [@Pena2015].
```
Comment (edzer) [inline _rmd/2017-03-08-sperrorest-update.Rmd]: which package? Is toolkit a synonym for the package? This vignette, do you mean this blog post?

File: _rmd/2017-03-08-sperrorest-update.Rmd [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+
+This vignette is based on code that Alex Brenning developed for his course on 'Environmental Statistics and GeoComputation' that he teaches in the [Geographic Information Science Master's program](http://www.geographie.uni-jena.de/Studium/Studierende/Studiengänge/Master+of+Science+Geoinformatik+.html) at Friedrich Schiller University Jena, Germany. Please take a look at our program and spread the word!
```
Comment (edzer) [inline _rmd/2017-03-08-sperrorest-update.Rmd]: I think that advertising is not appropriate here; the link also points to dominantly German material.

File: _rmd/2017-03-08-sperrorest-update.Rmd [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+    x
+  }
+  
+  pred <- predict(object, newdata = newdata)
+  if (!is.null(fac)) pred <- majority.filter(pred, newdata[,fac]) 
+  return(pred)
+}
+```
+
+Running `sperrorest()` takes some time here (Test machine: 3,2 Ghz Intel I-5 with 4 physical cores).
```
Comment (edzer) [inline _rmd/2017-03-08-sperrorest-update.Rmd]: I reran the script locally, and removed the reference to the test machine

**Comment by pat-s** [comment]
@edzer Addressed your suggested changes +  removed unused code blocks - ready for publishing?

**Comment by edzer** [comment]
Pls check, it's online now.

**Comment by pat-s** [comment]
@edzer thanks! 

The following reference is falsely rendered on [r-spatial.org](http://r-spatial.org/r/2017/03/13/sperrorest-update.html) although it is right in the [.md file](https://github.com/r-spatial/r-spatial.org/blob/gh-pages/_posts/2017-03-08-sperrorest-update.markdown). 
Could you check on this? 

> James, Gareth, Daniela Witten, Trevor Hastie, and Robert Tibshirani. 2013. An Introduction to Statistical Learning. Springer New York. doi:10.1007/978-1-4614-7138-7.

**Comment by edzer** [comment]
fixed.

**Pull request merged**